### PR TITLE
docs: Chapter00 to PT issue #51

### PR DESCRIPTION
**Pull request by victorescosta**
Created specific folder to PT translation.
issue #51

### CODE REVIEW

**Review by lewtun — COMMENTED** [review]
Thank you for this contribution @victorescosta ! I'll ask the Portugese speakers to give this a quick review, but in the meantime could you please add the `_toctree.yml` file to this PR with the following content:

```
- title: 0. Setup # <= Feel free to translate this if it also makes sense
  sections:
  - local: chapter0/1
    title: Introdução
```

**Comment by LincolnVS** [comment]
Hi, I revised it and found some words that could fit better, but everything is okay. Nice!

File: chapters/pt/capítulo0/1.mdx [hunk 1]
```diff
@@ -0,0 +1,111 @@
+# Introdução
```
[the remark this answers is not in the document]
Comment (victorescosta) [reply chapters/pt/capítulo0/1.mdx]: All done, can you check if it's everything ok @lewtun?

File: .github/workflows/build_documentation.yml [hunk 2]
```diff
@@ -14,6 +14,6 @@ jobs:
       package: course
       path_to_docs: course/chapters/
       additional_args: --not_python_module
-      languages: en es fr th tr
+      languages: en es fr pt th tr
```
Comment (lewtun) [inline .github/workflows/build_documentation.yml:17]: FYI I've added Portuguese to your PR so we can build the docs on the website :)

**Review by lewtun — APPROVED** [review]
Thank you @victorescosta - this looks great!

**Pull request merged**